lib/widget/switch.dart

```diff
@@ -50,28 +61,30 @@ class EnsembleSwitch extends StatefulWidget
     };
   }
 
-  void onToggle(bool newValue) {
-    setProperty('value', newValue);
+  void setSwitchValue(dynamic value) {
```

Comment:
Switch and TripleSwitch's value should be succinct. The former takes in true/false, the latter 3 states. There should be no guessing like this, so please refactor so their inputs are separate. The schema should reflect tis too

Reply:
Added `onToggle` in those widgets with their state as input parameter